site-src/faq.md

```diff
@@ -66,5 +66,17 @@
   AppProtocol depend on Kubernetes 1.18 (opt-in) or 1.19 (on by default). 
   There are not any other exceptions to the 1.16+ guideline right now.
 
+* **Q: Is SSL Passthrough supported?**
+  A: SSL Passthrough (wherein the Gateway routes traffic with the [Transport
+  Layer Security (TLS)][tls] encryption _intact_ to a backend service instead of
+  terminating it) is not currently a supported routing option when using the
+  [HTTPRoute][routes] API. If you need this functionality it is supported by
+  the [TLSRoute][tlsroute] API. Also see the [TLS Guide][tlsguide] for more
+  details about passthrough and other TLS configurations.
```

Comment:
I'd recommend framing it rather as "Yes, it is supported using TLSRoute. It is not supported in HTTPRoute because it is not possible to route based on HTTP metadata without terminating TLS."
This educates our users. The current wording makes me feel "something is possible but is not supported in the API".

Reply:
Sounds good, 3dc2ed4bf35b970b97193da78c490129aa1e4d9a changes the language lmkwyt